packages/wrangler/src/index.tsx

```diff
@@ -1233,6 +1249,12 @@ export async function main(argv: string[]): Promise<void> {
               );
             }
 
+            if (args.local) {
```

Comment:
This isn't true, we do support local mode for this command. Did you mean to put this in wrangler tail?

Reply:
Oops! Sorry.